docs/cog_guides/alias.rst

```diff
@@ -0,0 +1,270 @@
+.. _alias:
+
+=====
+Alias
+=====
+
+This is the cog guide for the alias cog. You will
+find detailed docs about the usage and the commands.
+
+``[p]`` is considered as your prefix.
+
+.. note:: To use this cog, load it by typing this::
+
+        [p]load alias
+
+.. _alias-usage:
+
+-----
+Usage
+-----
+
+This cog allows to create shortcuts for commands.
+
+You can do things like this:
+
+.. code-block:: none
+
+    [p]cleanup messages 42
+    equals to
```

Comment:
Instead of saying "equals to", explain better how the top command can be aliased into the bottom command.